boards/arm64/imx8mm_evk/imx8mm_evk_jailhouse.yaml

```diff
@@ -0,0 +1,12 @@
+identifier: imx8mm_evk_jailhouse
+name: NXP i.MX8M Mini EVK A53 Jailhouse Inmate Cell
```

Comment:
can you add the info that is now in the commit message in the README?

Reply:
Add info to doc/index.rst.